web/views.py

```diff
@@ -290,11 +292,11 @@ def get_initial(self):
     def get_success_url(self):
         return reverse("web:studies-list")
 
-    def completed_consent_frame(self, studies, child):
+    def studies_without_completed_consent_frame(self, studies, child):
         return [
             r.study
             for r in Response.objects.filter(
-                study__in=studies, child=child, completed_consent_frame=True
+                study__in=studies, child=child, completed_consent_frame=False
             ).distinct("study_id")
         ]
```

Comment:
this is unfortunately not quite the logic we want: what we need is studies for which there is *no* response with a completed consent frame. This is fetching studies where there are responses that don't have completed consent frames. More concretely, say that there are the following studies:
* Study 1: baby has no responses
* Study 2: baby started study, but didn't complete consent frame
* Study 3: baby started study but didn't complete consent frame one time, then tried again and completed consent frame
* Study 4: baby started study and completed consent frame

When we hide studies we've done before, we want to show studies 1 and 2. I think this logic is going to show studies 2 and 3 instead.

Reply:
Hide these:
1. Any complete consent frame flags on any responses.  There could be more than one response. 
2. Show studies that have no responses.

Reply:
I have updated the query in 1d23c60.